src/test/scala/io/iohk/ethereum/transactions/PendingTransactionsManagerSpec.scala

```diff
@@ -115,14 +115,17 @@ class PendingTransactionsManagerSpec extends FlatSpec with Matchers with ScalaFu
     pendingTransactionsManager ! AddOrOverrideTransaction(firstTx)
     peerManager.expectMsg(PeerManagerActor.GetPeers)
     peerManager.reply(Peers(Map(peer1 -> Handshaked)))
+    Thread.sleep(Timeouts.normalTimeout.toMillis)
```

Comment:
no I waited `1 *  Timeouts.normalTimeout` but I tested it now and after waiting `3 * Timeouts.normalTimeout` it fails